### PR TITLE
Add OpenJDK installation guide

### PR DESCRIPTION
**Pull request by journey2w**
### Change Description ###

Add OpenJDK installation guide.

### CODE REVIEW

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/527

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/527

File: docs/application/tizen-studio/setup/openjdk.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+4. Click **New...** in the **User variables for \<user name\>** section.
+    > **Note**
+    >
+    > If your `JAVA_HOME` variable is already created, select it and click **Edit...**.
+
+5. Enter `JAVA_HOME` in the variable name field and the JDK directory path, for example, `C:\Users\user\Desktop\jdk-10.0.2`, in the Variable value field.
+
+6. Click **OK**. The Environment Variables window appears.
+
+7. If you cannot find `JAVA_HOME` in **System variables** section, add the variable repeat step 4, 5, and 6.
```
[the remark this answers is not in the document]
Comment (mijin85cho) [reply docs/application/tizen-studio/setup/openjdk.md]: It looks good to me.

**Comment by huytaduc** [comment]
checkout the webpage https://docs.stg.tizen.org/staging/527

**Comment by mijin80lee** [comment]
TD Sync Completed.

**Pull request merged**